src/Common/src/System/Net/Http/HttpHandlerDefaults.cs

```diff
@@ -16,5 +16,6 @@ internal static class HttpHandlerDefaults
         public const bool DefaultPreAuthenticate = false;
         public const ClientCertificateOption DefaultClientCertificateOption = ClientCertificateOption.Manual;
         public const bool DefaultUseProxy = true;
+        public readonly static TimeSpan DefaultConnectTimeout = TimeSpan.FromSeconds(60);
```

Comment:
This looks like it'll add a .cctor to this type, which didn't have one before.  I wonder if it's worth it.  What about making it a property instead:

```csharp
public static TimeSpan DefaultConnectTimeout => TimeSpan.FromSeconds(60);
```